layouts/partials/navbar.html

```diff
@@ -26,7 +26,7 @@
 
       <div class="navbar-end">
         {{- if .Site.Params.search }}
-        <button class="search-button" title="Search (Ctrl+k)">
+        <button class="search-button" title="Search (Ctrl/Cmd+K)">
```

Comment:
While this should work, to truly achieve an implementation where it displays just "Cmd/⌘" on macOS and "Ctrl" on other platforms, I wrote an event listener for this – however, I wish to confirm if that's okay to add before I get towards committing/pushing those changes

Reply:
Actually, it's quite a small change. I added it to the existing event listener, so it should be easy enough for a review!